tools/isobuild/bundler.js

```diff
@@ -575,6 +575,14 @@ class File {
     // file is not intended to be served over HTTP).
     this.url = null;
 
+    // A prefix that will be prepended to this.url.
+    if (options.arch.startsWith("web.")) {
+      this.urlPrefix = "/__" +
+        options.arch.split(".").slice(1).join(".");
+    } else {
+      this.urlPrefix = "";
+    }
```

Comment:
⚠️ 🚨 Static resource URLs now begin with `/__browser/...` or `/__browser.legacy/...`, which may be a breaking change for code that attempted to load these URLs without consulting the manifest. 🚨 ⚠️ 

The good news is that this code is now the single authoritative source of those `/__browser/...` and `/__browser.legacy/...` URL prefixes, which is why very little other code (besides some tests) had to change to accommodate the new URLs.